plugins/dfsound/externals.h

```diff
@@ -115,6 +115,8 @@ typedef struct
  int               iRightVolume;                       // right volume
  ADSRInfoEx        ADSRX;
  int               iRawPitch;                          // raw pitch (0...3fff)
+ unsigned char	   IgnoreLoop;
+ unsigned char*	   pStart;

```

Comment:
I hope this can be avoided. Everything multiplies by 24 here so you waste 96 bytes on ARM here, which hurts slow devices with tiny caches. See my other comment...